packages/server/src/parse-chokidar-rules-from-gitignore.ts

```diff
@@ -0,0 +1,55 @@
+import parseGitignore from 'parse-gitignore'
+import fs from 'fs'
+import {resolve} from 'path'
+import _ from 'lodash'
+
+function getAllGitIgnores(rootFolder: string) {
+  function getGitIgnoresRecursively(folder: string, prefix: string): {gitIgnore: string; prefix: string}[] {
```

Comment:
Any reason we can't just fast-glob for gitignore files?

Reply:
Didn't know about fast-glob, will use 👍🏻